db/seeds.rb

```diff
@@ -5,3 +5,32 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Passenger.delete_all
+Booking.delete_all
+
+Airport.delete_all   
+lag = Airport.create(city: "LAG", name: "Muritala Mohammed Airport Lagos")
+abj = Airport.create(city: "ABJ", name: "Nnamdi Azikiwe Airport Abuja")
+phc = Airport.create(city: "PHC", name: "Port Harcourt International Airport")
+kad = Airport.create(city: "KAD", name: "Kaduna International Airport")
+enu= Airport.create(city: "ENU", name: "Enugu International Airport")
+
+
+Flight.delete_all
+
+
+Airport.all.each do |airport1|
+  Airport.all.each do |airport2|
+    unless airport1 == airport2
+      Flight.create(dept_id: airport1.id, arr_id: airport2.id, flight_date: Time.now, price: $3000)
```

Comment:
Line is too long. [99/80]<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.